aries-site/src/pages/components/checkbox.mdx

```diff
@@ -36,8 +37,22 @@ Checked and unchecked states refer to when an individual CheckBox is checked or
 
 ### CheckBox outside of FormField
 
+When using CheckBox outside of Formfield, it does not have a border or label. The hover indentifier will also be on the CheckBox itself.
+Here are some common use cases for when a CheckBox can be used outside of a FormField:
+1. When there is progressive disclosure: i.e. clicking on the checkbox will reveal related options that wouldn't have been displayed unless the user checked the box.
+2. There is no additional information or help text necessary for the user to understand what is being asked of them.
+3. Allowing a user to opt-in or acknowledge a disclaimer, such as a privacy policy or terms and conditions.
+
```

Comment:
Can I use a checkbox outside a form field for selecting an item from a list?

Reply:
umm not sure I can picture what you are referring to?

Reply:
- Allowing an item to be selected within a list, table row, or drop down.
☝️ do we use `checkbox` in a list? In a DataTable checkbox is built in so just trying to understand where this case would be used cant visualize it